system/CLI/CLI.php

```diff
@@ -220,17 +221,26 @@ public static function input(string $prefix = null): string
 	 *
 	 * @param string       $field      Output "field" question
 	 * @param string|array $options    String to a default value, array to a list of options (the first option will be the default value)
-	 * @param string       $validation Validation rules
+	 * @param string|array $validation Validation rules
 	 *
 	 * @return string The user input
 	 *
 	 * @codeCoverageIgnore
 	 */
-	public static function prompt(string $field, $options = null, string $validation = null): string
+	public static function prompt(string $field, $options = null, $validation = null): string
 	{
 		$extraOutput = '';
 		$default     = '';
 
+		if ($validation && ! is_array($validation) && ! is_string($validation))
+		{
+			throw new InvalidArgumentException('$rules can only be of type string|array');
+		}
```

Comment:
Add new line

Reply:
@element-code Rector is complaining. You must've added trailing whitespace while adding the new line.

Reply:
Yeah, thanks to the Github Code-Editor 🙃